ledger/participant-integration-api/src/test/suite/scala/platform/apiserver/meteringreport/HmacSha256Spec.scala

```diff
@@ -30,14 +30,15 @@ class HmacSha256Spec extends AnyWordSpec with Matchers {
     "compute MAC" in {
       val expected = "uFfrKWtNvoMl-GdCBrotl33cTFOqLeF8EjaooomUKOw="
       val key = MeteringReportKey.communityKey()
-      val Right(mac) = HmacSha256.compute(key, "some message".getBytes(StandardCharsets.UTF_8))
-      val actual = toBase64(mac)
-      actual shouldBe expected
+      HmacSha256.compute(key, "some message".getBytes(StandardCharsets.UTF_8)).map { mac =>
+        val actual = toBase64(mac)
+        actual shouldBe expected
+      }
```

Comment:
if the value you get here is `left` it would not fail... I suggest mixing in `EitherValues`:
https://www.scalatest.org/user_guide/using_EitherValues

Reply:
:+1: